src/envoy/http/service_control/http_call.cc

```diff
@@ -101,8 +102,13 @@ class HttpCallImpl : public HttpCall,
 
         ENVOY_LOG(debug, "http call response status code: {}, body: {}",
```

Comment:
Slightly unrelated to this PR, but can you move this log line to be before `attemptRetry`? Otherwise, the logs will be out of order when the stack unwinds on retries.

Reply:
Done